Meta/Lagom/CMakeLists.txt

```diff
@@ -256,18 +262,24 @@ if (NOT TARGET all_generated)
     add_custom_target(all_generated)
 endif()
 
+# Only enable gcc extensions for LibCore, LibMain & LibTimeZone under cygwin
+if (CYGWIN)
+    set(CMAKE_CXX_EXTENSIONS ON)
+endif()
+
 # AK/LibCore
 # Note: AK is included in LibCore for the host build instead of LibC per the target build
 file(GLOB AK_SOURCES CONFIGURE_DEPENDS "../../AK/*.cpp")
 file(GLOB LIBCORE_SOURCES CONFIGURE_DEPENDS "../../Userland/Libraries/LibCore/*.cpp")
-if (ANDROID)
+if (ANDROID OR CYGWIN)
     list(REMOVE_ITEM LIBCORE_SOURCES "${CMAKE_CURRENT_LIST_DIR}/../../Userland/Libraries/LibCore/Account.cpp")
+    list(REMOVE_ITEM LIBCORE_SOURCES "${CMAKE_CURRENT_LIST_DIR}/../../Userland/Libraries/LibCore/Group.cpp")
```

Comment:
Add some documentations about which features of lagom are not available on Windows.

Reply:
Same as on Android, you can see what's missing by reading the code. #14662 will take care of implementing the rest of the libraries. After that it makes sense to spend time on documentation